clustered/integration-test/src/test/java/org/ehcache/clustered/management/AbstractClusteringManagementTest.java

```diff
@@ -87,8 +90,10 @@
   protected static Connection managementConnection;
 
   @ClassRule
-  public static Cluster CLUSTER = newCluster().in(new File("build/cluster"))
-                                              .withServiceFragment(RESOURCE_CONFIG).build();
+  public static Cluster CLUSTER = newCluster()
+    .in(new File("build/cluster"))
+    .withClientReconnectWindowTime(5)
+    .withServiceFragment(RESOURCE_CONFIG).build();
```

Comment:
Now that we have a nice line wrapping, please put the `build()` on a new line as well :-)